public/scripts/utils.js

```diff
@@ -17,3 +17,11 @@ export function $try(func) {
 export function reformatDatetime(dtStr) {
   return new Date(dtStr).toString();
 }
+
+export function downloadCSV(data, filename) {
```

Comment:
@stefanv I thought about this, and we would either have to a) serve a CSV version of all predictions in the database all the time, or b) generate it when the user clicks Download, in which case we'd still need a utility function like this to dynamically create and click the link, or redirect to the CSV URL, after it's been generated.